products/installation.yaml

```diff
@@ -69,8 +69,8 @@ products:
     marin3r:
         channel: stable
         installFrom: index
-        package: marin3r-operator
-        index: quay.io/integreatly/marin3r-index:v0.13.1
+        package: marin3r
+        index: quay.io/integreatly/marin3r-operator:v0.13.2-index
```

Comment:
I think the correct URL here is:
quay.io/integreatly/marin3r-index:v0.13.2

Reply:
done. Thank you for comment